### PR TITLE
DeepEdit App for Vertebra

### PR DESCRIPTION
**Pull request by diazandr3s**
- MONAI Label App created for vertebra segmentation using the VerSe2020 dataset (https://osf.io/b2wxj/). App to show during the Slicer workshop

Signed-off-by: Andres Diaz-Pinto <diazandr3s@gmail.com>

### CODE REVIEW

**Comment by diazandr3s** [comment]
Here are the pretrained weights for deepedit vertebra App: https://emckclac-my.sharepoint.com/:u:/g/personal/k2039747_kcl_ac_uk/ESBNIt7TmmRHuoFGNr_ZKBIBMbTItGeveKPJbrAQT-cyPA?e=5Yo9FL

**Comment by SachidanandAlle** [comment]
please delete this file sample-apps/deepedit_vertebra/.DS_Store

File: sample-apps/deepedit_vertebra/main.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        }
+
+        strategies = {
+            "random": Random(),
+            "first": MyStrategy(),
+            "tta": TTA(path, self.network),
+        }
+
+        resources = [
+            (self.pretrained_model, "https://www.dropbox.com/s/1y4j9o12zadauhx/deepedit_vertebra.pt?dl=1"),
```
Comment (SachidanandAlle) [inline sample-apps/deepedit_vertebra/main.py:66]: please use https://www.dropbox.com/s/a2fqn2hzgdxkrth/deepedit_vertebra.pt?dl=1

**Comment by SachidanandAlle** [comment]
Add model/.gitignore file

**Comment by SachidanandAlle** [comment]
add the sample app entry so that users can download it as part of CLI
https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/main.py#L152

**Review by SachidanandAlle — CHANGES_REQUESTED** [review]
Please fix the comments as mentioned in PR

**Review by SachidanandAlle — APPROVED** [review]

File: monailabel/main.py [hunk 2]
```diff
@@ -150,6 +150,7 @@ def run_apps(args):
         "deepgrow_spleen": f"{sample_apps_uri}/deepgrow_spleen",
         "segmentation_left_atrium": f"{sample_apps_uri}/segmentation_left_atrium",
         "segmentation_spleen": f"{sample_apps_uri}/segmentation_spleen",
+        "segmentation_vertebra": f"{sample_apps_uri}/segmentation_vertebra",
```
Comment (SachidanandAlle) [inline monailabel/main.py:153]: it has to be deepedit_vertebra

**Review by SachidanandAlle — CHANGES_REQUESTED** [review]
it has to be deepedit_vertebra in main.py

**Comment by diazandr3s** [comment]
> it has to be deepedit_vertebra in main.py

Just changed it. Thanks!

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**